src/main.ts

```diff
@@ -1,12 +1,29 @@
+import { DIPDeposit } from './common';
+import { Poller } from './poller';
+import { Router } from './router';
+import { Scalper } from './scalper';
+
 function main() {
-    // Start the scalper
+    // Create a scalper
+
+    // Create a router
+    const router: Router = new Router(
+        (deposit: DIPDeposit) => { console.log(deposit); },
+        (deposit: DIPDeposit) => { console.log(deposit); }
```

Comment:
duped?

Reply:
Placeholder. One is for the MM, one is for scalper